ipa/Cargo.toml

```diff
@@ -0,0 +1,141 @@
+[package]
+name = "ipa"
+version = "0.1.0"
+rust-version = "1.64.0"
+edition = "2021"
+
+[features]
+# by default remove all TRACE, DEBUG spans from release builds
+default = ["web-app", "tracing/max_level_trace", "tracing/release_max_level_info"]
```

Comment:
Removed "in-memory-infra" from `default`. Dependent external crates will specify either `in-memory-infra` or `real-world-infra` in their own Cargo.toml.

Reply:
this broke most of our unit tests - I checked the CI and we only run 55 tests because of it

Reply:
oops, thanks for catching. Fixed.